test/automated-suite.js

```diff
@@ -24,7 +24,7 @@
 
 require('chai').should();
 const path = require('path');
-const mochaUtils = require('../node_modules/sw-testing-helpers/src/mocha/utils.js');
+const mochaUtils = require('sw-testing-helpers').mochaUtils;
```

Comment:
FYI - what you were doing before could also have been done cleaner as `require('sw-testing-helpers/src/mocha/utils')`. Require will find things relative to installed modules. Exporting the things that are needed is even better though, of course!